packages/harmony/src/components/comments/IconText/IconText.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { Divider, Flex } from 'components/layout'
+import { IconHeart, IconPin, IconStar, IconTipping, IconTrophy } from 'icons'
+
+import { IconText } from './IconText'
+
+const meta: Meta<typeof IconText> = {
+  title: 'Components/Comments/IconText [beta]',
+  component: IconText
+}
+
+export default meta
+
+type Story = StoryObj<typeof IconText>
+
+export const Default: Story = {
+  render: () => (
+    <Flex direction='column' gap='l' p='s'>
+      <IconText text='Test Text' />
+      <IconText text='Test Text' color='accent' />
+      <IconText text='Test Text' color='active' />
+
+      <Divider />
+
+      <IconText icons={[{ icon: IconPin }]} text='Pinned By Artist' />
+      <IconText
+        icons={[{ icon: IconHeart, color: 'active' }]}
+        text='Liked By Artist'
+      />
+      <IconText
+        icons={[{ icon: IconPin }, { icon: IconHeart, color: 'active' }]}
+        text='Liked By Artist'
+      />
+
+      <Divider />
+
+      <IconText
```

Comment:
Do we want the text part to be children prop?

Reply:
yea child prop makes sense to work like the Text component